tests/unit/wait-for-coroutine-test.ts

```diff
@@ -0,0 +1,115 @@
+import { module, test } from 'qunit';
+import { _reset, waitForCoroutine, getPendingWaiterState } from 'ember-test-waiters';
+import MockStableError, { resetError, overrideError } from './utils/mock-stable-error';
+import { task } from 'ember-concurrency-decorators';
+import EmberObject, { get } from '@ember/object';
+import { task as taskFn } from 'ember-concurrency';
+import { DEBUG } from '@glimmer/env';
+
+if (DEBUG) {
```

Comment:
Would you mind moving the guard _inside_ the module?

Something like:

```js
module('wait-for-coroutine', function(hooks) {
  if (!DEBUG) {
    return;
  }

  // ...existing code...
});
```

Reply:
Sure. Curious -- why do you prefer that?